cdc/metrics/grafana/tikv-cdc.json

```diff
@@ -6496,100 +6496,6 @@
             "alignLevel": null
           }
         },
-        {
```

Comment:
Removing a panel will leave a blank space, may need move all other panels or modify the metric json in next pr?

Reply:
yesh, it will leave a blank space. I add it back, and remove it in next pr. Thanks